.github/workflows/update-load-generator-image.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow uses GHA to build a Java project with Gradle 'load-generator', and build and push multi-arch docker image of this project to the Amazon ECR.
+
+name: Update Load-Generator Image
+
+# Should execute only one workflow run at a time
+concurrency:
+  group: build-and-push-load-generator-image
+
+on:
+  workflow_dispatch:
+  push:
+    branches: ['terraform']
+
+    # List of paths that should be considered when triggering the workflow
+    paths: ['load-generator/**']
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  create-and-push-load-generator-image-on-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.TEST_FW_ASSUMABLE_ROLE_ARN }}
+          aws-region: us-west-2
+          role-duration-seconds: 7200
+      - name: Login to Amazon ECR Public
+        id: login-ecr-public
+        uses: aws-actions/amazon-ecr-login@v1
+        with:
+          registry-type: public
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build image with jib
+        uses: burrunan/gradle-cache-action@v1
```

Comment:
I'm not familiar with this action. It also does not appear to be actively maintained. Have you considered using the marketplace action that the java-instrumentation uses? can be seen [here](https://github.com/aws-observability/aws-otel-java-instrumentation/blob/22af6a0e21b980aeacef711e2fe4db74ef5d1a6b/.github/workflows/main-build.yml#L49).

Reply:
Updated.